server/chat.ts

```diff
@@ -971,44 +965,39 @@ export class CommandContext extends MessageContext {
 
 		if (!user.named) {
 			connection.popup(this.tr(`You must choose a name before you can talk.`));
-			return null;
+			throw new Chat.ErrorMessage(this.tr(`You must choose a name before you can talk.`));
```

Comment:
It looks like you're showing the error twice?

Reply:
Intentionally, so it cancels / cancels commands. I guess i could not throw a message? But that's ugly..

Reply:
I don't know what you mean.

Reply:
(We discussed this on Discord and figured it out; we decided the popup wasn't necessary anymore.)